packages/frontend/src/pages/Inbox/Inbox.tsx

```diff
@@ -123,6 +119,15 @@ export const Inbox = ({ viewType }: InboxProps) => {
     }
   }, [showingSearchResults]);
 
+  // biome-ignore lint/correctness/useExhaustiveDependencies: Full control of what triggers this code is needed
+  const itemsToDisplay = useMemo(() => {
+    return filterDialogs(dataSource, activeFilters, format);
+  }, [dataSource, activeFilters]);
+
+  const filterBarSettings = getFilterBarSettings(dataSource, itemsToDisplay, format).filter(
+    (setting) => setting.options.length > 1 || setting.id === 'updated',
```

Comment:
hvorfor kun for `updated`? 

det må være noe sånt som dette, tror jeg:

`setting.options.length > 1 || typeof activeFilters.find((filter) => filter.id === setting.id) !== 'undefined'`